python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -301,10 +301,10 @@ class LogisticRegression(Base, ClassifierMixin):
             )
 
         if self.fit_intercept:
-            self.coef_ = self.qn.coef_[0:-1]
-            self.intercept_ = self.qn.coef_[-1]
+            self.coef_ = self.solver_model.coef_[0:-1]
+            self.intercept_ = self.solver_model.coef_[-1]
         else:
-            self.coef_ = self.qn.coef_
+            self.coef_ = self.solver_model.coef_
```

Comment:
Same as the previous

Reply:
Yeah, you're right. Although logistic regression uses this for logging and it might be messy to have these conditions there. 

Do we want to remove the logging from logistic regression and place it in QN?

Reply:
the logging in logistic should just work even if we remove this from here, no? for example, in the following lines of this 
```python
     if logger.should_log_for(logger.level_trace):
            logger.trace(self.verb_prefix + "Coefficients: " +
                         str(self.coef_.to_output("cupy")))
``` 
self.coef_ would still get picked up from the `qn` class, so this should still work even while removing this

Reply:
Ah yes, you're right. I'll make this change!